### ISSUE TITLE
Histogram sort order

### ISSUE THREAD

**Issue by patrickjohncyh** [LangProc/langproc-2017-lab#56]
The spec specifies that the primary sort order as : 

"primary sort order: the number of times it occurs, from least to most."

However, the reference for test 6 orders the histogram from most to least.

**Comment by m8pple** [comment]
Yes, you're right - the reference output relies on the sort order defined in `histogram_main.cpp`,
which is actually the other way round (the same as for test 6).

I'll stick with the version in the code and modify the text, as it is intended that people don't
need to deal with that part.

I usually try to modify lab exercises each year at a surface level to discourage temporal migration
of solutions/code - I think this is a consequence of that.

Thanks for spotting that.
